cmd/otelcontribcol/builder-config.yaml

```diff
@@ -60,6 +60,7 @@ exporters:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.83.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerthrifthttpexporter v0.83.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.83.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kineticaexporter v0.83.0
```

Comment:
Lets modify this command during a later PR.  The component is not functional in its current state so it doesn't need to be added to this build manifest yet.

Reply:
This line has been removed.